app/src/main/java/com/duckduckgo/app/statistics/api/StatisticsRequester.kt

```diff
@@ -70,31 +70,52 @@ class StatisticsRequester(
     private fun storedAtbFormatNeedsCorrecting(storedAtb: Atb): Boolean = storedAtb.version.endsWith(LEGACY_ATB_FORMAT_SUFFIX)
 
     @SuppressLint("CheckResult")
-    override fun refreshRetentionAtb() {
+    override fun refreshSearchRetentionAtb() {
 
         val atb = store.atb
-        val retentionAtb = store.retentionAtb
 
-        if (atb == null || retentionAtb == null) {
+        if (atb == null) {
             initializeAtb()
             return
         }
 
+        val fullAtb = atb.formatWithVariant(variantManager.getVariant())
+        var retentionAtb = store.searchRetentionAtb ?: atb.version
+
+        service.updateSearchAtb(fullAtb, retentionAtb)
+            .subscribeOn(Schedulers.io())
+            .subscribe({
+                Timber.v("Search atb refresh succeeded")
```

Comment:
Can we make this log statement more useful for debugging by adding in the actual value? including `it.version` in the log